README.md

```diff
@@ -89,33 +93,31 @@ variables:
 - **AccessChecker**: As mentioned above, access-checkers can be provided as
   plugins and easily swapped. Each access-checker has a name (see
   [plugins](plugins) for details) and `ACCESS_CHECKER` variable should be set to
-  this name. For example, the two plugins that are provided in this repository,
-  can be selected by either of:
+  this name. 
+  
+  For example, the two plugins that are provided in this repository, can be selected by either of:
 
   ```shell
   export ACCESS_CHECKER=list
   export ACCESS_CHECKER=patient
   ```
+  NOTE: The above are just samples provided to demonstrate the concept of a pluggable access-checker. Custom access-checkers can be implemented to support common patterns such as use of `Location`, `Care Team` or `General Practitioner` to return the conceptual list of Patients for an authenticated user
```

Comment:
nit: Isn't it better to make links to FHIR resources when we talk about `Location`, `CareTeam`, etc?